src/libraries/Microsoft.PowerFx.Core/Texl/ViewFinderVisitor.cs

```diff
@@ -34,7 +35,12 @@ public override void PostVisit(DottedNameNode node)
         }
 
         public override void Visit(FirstNameNode node)
-        {
+        {

```

Comment:
curious - why are Views special cased (relative to tables or other things)?

Reply:
Answered offline.